qualtran/qir_example.ipynb

```diff
@@ -0,0 +1,685 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 42,
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pyqir\n",
```

Comment:
We'll have to add `pyqir` to the requirements file and recompile dependencies using `dev_tools/requirements/re-pip-compile.sh`